### PR TITLE
Kinetic devel

### PR DESCRIPTION
**Pull request by woodaaron**
Update to Godel allowing it to work with an Ensenso camera and to operate on Ubuntu 16.04 LTS/ROS Kinetic.

### CODE REVIEW

File: godel_robots/abb/godel_irb2400/godel_irb2400_support/launch/irb2400_blending.launch [hunk 1]
```diff
@@ -37,15 +48,33 @@
       <rosparam command="load" file="$(find godel_irb2400_support)/config/point_cloud_descriptions.yaml"/>
     </node>
   </group>
+
+  <!-- If the user uses the real sensor, launch the ensenso publisher node -->
+  <group unless="$(arg sim_sensor)">
+    <remap from="/depth/points" to="/sensor_point_cloud"/>
+    <param name="camera_frame_id" type="string" value="tool0" />
+    <node name="ensenso_publisher_node" pkg="ensenso_publisher" type="ensenso_publisher_node" output="screen" required="true">
```
Comment (VictorLamoine) [inline godel_robots/abb/godel_irb2400/godel_irb2400_support/launch/irb2400_blending.launch:56]: Is it this package? https://github.com/Jmeyer1292/ensenso_publisher

You forgot to require the package dependency in the CMakeLists.txt and package.xml

Comment (woodaaron) [reply godel_robots/abb/godel_irb2400/godel_irb2400_support/launch/irb2400_blending.launch:56]: That's the package. Specifically, you're talking about CMakeLists.txt and package.xml for the godel_irb2400_support package?

Comment (gavanderhoorn) [reply godel_robots/abb/godel_irb2400/godel_irb2400_support/launch/irb2400_blending.launch:56]: This is just a runtime dependency, you don't add those to `CMakeLists.txt`.

Comment (VictorLamoine) [reply godel_robots/abb/godel_irb2400/godel_irb2400_support/launch/irb2400_blending.launch:56]: My comment was wrong/unclear, I created an issue to detail what I think is wrong:
https://github.com/Jmeyer1292/ensenso_publisher/issues/1

**Comment by Jmeyer1292** [comment]
@woodaaron Per our conversation, split up into more PRs starting at #121

**Pull request closed without merging**